appyx-components/demos/android/src/main/kotlin/com/bumble/appyx/components/demos/cards/DatingCards.kt

```diff
@@ -41,57 +35,43 @@ fun DatingCards(modifier: Modifier = Modifier) {
                 },
                 savedStateMap = null
             ),
-            motionController = { CardsMotionController(it)  },
+            motionController = { CardsMotionController(it) },
             gestureFactory = { CardsMotionController.Gestures(it) },
             animateSettle = true
         )
     }
 
     InteractionModelSetup(cards)
 
-    Children(
+    DraggableChildren(
         modifier = modifier
             .fillMaxSize()
             .background(appyx_dark)
             .padding(16.dp),
+        screenWidthPx = (LocalConfiguration.current.screenWidthDp * LocalDensity.current.density).roundToInt(),
+        screenHeightPx = (LocalConfiguration.current.screenHeightDp * LocalDensity.current.density).roundToInt(),
         interactionModel = cards,
-        element = {
-            ElementWrapper(
-                elementUiModel = it,
-                modifier = Modifier
-                    .fillMaxSize()
-                    .pointerInput(it.element.id) {
-                        detectDragGestures(
-                            onDragStart = { position -> cards.onStartDrag(position) },
-                            onDrag = { change, dragAmount ->
-                                change.consume()
-                                cards.onDrag(dragAmount, density)
-                            },
-                            onDragEnd = {
-                                cards.onDragEnd(
-                                    completionThreshold = 0.15f,
-                                    completeGestureSpec = spring(stiffness = Spring.StiffnessLow),
-                                    revertGestureSpec = spring(stiffness = Spring.StiffnessMedium)
-                                )
-                            }
-                        )
-                    }
+        onDragStart = { position -> cards.onStartDrag(position) },
+        onDrag = { change, dragAmount, _ ->
+            change.consume()
+            cards.onDrag(dragAmount, density)
+            true
+        },
+        onDragEnd = {
+            cards.onDragEnd(
+                completionThreshold = 0.15f,
+                completeGestureSpec = spring(stiffness = Spring.StiffnessLow),
+                revertGestureSpec = spring(stiffness = Spring.StiffnessMedium)
             )
```

Comment:
Maybe even this can be moved to automatic code under the hood, and then we only need to pass an optional `GestureSettleConfig` object that holds these 3 params.